<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskPanelCfdDynamicMeshRefinement</class>
 <widget class="QWidget" name="TaskPanelCfdDynamicMeshRefinement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>1078</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dynamic Mesh refinement</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="dynamicMeshFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout_11">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item row="1" column="2">
       <widget class="QDoubleSpinBox" name="input_refinement_interval">
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>1.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_refinement_interval">
        <property name="text">
         <string>Refinement interval</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>No buffer layers</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QDoubleSpinBox" name="doubleSpinBox">
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="thicknessLabel">
        <property name="text">
         <string>Max cells</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QDoubleSpinBox" name="if_edgerefinement">
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>99.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Max refinement level</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="Gui::InputField" name="thicknessInput">
        <property name="singleStep" stdset="0">
         <double>1.000000000000000</double>
        </property>
        <property name="unit" stdset="0">
         <string notr="true">mm</string>
        </property>
        <property name="format" stdset="0">
         <string>g</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Mesh</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="triggerFieldFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="extrusionLayout">
      <item row="2" column="1">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Lower refinement level</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Upper refinement level</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Refinement field</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="Gui::InputField" name="if_lower_refinement">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string extracomment="Select 0 to use default value"/>
        </property>
        <property name="text">
         <string>0.0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="singleStep" stdset="0">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum" stdset="0">
         <double>1000000000.000000000000000</double>
        </property>
        <property name="unit" stdset="0">
         <string notr="true">mm</string>
        </property>
        <property name="format" stdset="0">
         <string>g</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="2">
       <widget class="Gui::InputField" name="if_upper_refinement">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string extracomment="Select 0 to use default value"/>
        </property>
        <property name="text">
         <string>0.0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="singleStep" stdset="0">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum" stdset="0">
         <double>1000000000.000000000000000</double>
        </property>
        <property name="unit" stdset="0">
         <string notr="true">mm</string>
        </property>
        <property name="format" stdset="0">
         <string>g</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Trigger field</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="2">
       <widget class="QComboBox" name="cb_refinement_field">
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="surfaceOrInernalVolume">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QCheckBox" name="cb_write_refinement_volscalarfield">
        <property name="text">
         <string>Write refinement fields after each write interval</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::InputField</class>
   <extends>QLineEdit</extends>
   <header>Gui/InputField.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>input_refinement_interval</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
